propertysuggester/analyzer/rule.py

```diff
@@ -22,3 +22,6 @@ def __eq__(self, other):
 
     def __str__(self):
         return str(self.__dict__)
+
+    def __repr__(self):
```

Comment:
What is this?

Reply:
This is a good explanation: https://stackoverflow.com/questions/1436703/difference-between-str-and-repr-in-python

Reply:
This sounds like you should remove the `__str__` duplicates: *"if `__repr__` is defined, and `__str__` is not, the object will behave as though `__str__`=`__repr__`."*

Reply:
Okay, removed it.